site/earlgrey-diagram/src/earlgrey.html

```diff
@@ -0,0 +1,365 @@
+      <lowrisc-block class="subdomain-badge b48mhz" pos="0.5 0.5"></lowrisc-block>
+      <lowrisc-block class="subdomain-badge b200khz" pos="0.5 1.75"></lowrisc-block>
+      Analog Sensor Top
+    </lowrisc-block>
+
+    <lowrisc-block id="clock-speeds" pos="1 40">
+      <lowrisc-block class="subdomain-badge b100mhz" pos="0 0 6 2">
+        ~100 MHz
+      </lowrisc-block>
+
+      <lowrisc-block class="subdomain-badge b96mhz" pos="7 0 6 2">
+        96 MHz
+      </lowrisc-block>
+  
+      <lowrisc-block class="subdomain-badge b48mhz" pos="14 0 6 2">
+        48 MHz
+      </lowrisc-block>
+  
+      <lowrisc-block class="subdomain-badge b24mhz" pos="21 0 6 2">
+        24 MHz
+      </lowrisc-block>
+  
+      <lowrisc-block class="subdomain-badge b200khz" pos="28 0 6 2">
+        200 kHz
+      </lowrisc-block>
+  
+      <lowrisc-block class="subdomain-badge blogic" pos="35 0 6 2">
+        Logic Only
+      </lowrisc-block>
+    </lowrisc-block>
+  </lowrisc-block>
+</lowrisc-block>
+
+<script type="module" id="hydrate">
+  import "../src/hydrate.js";
+</script>
+
+<!--
+<style>
+  #theme {
+    display: flex;
+    padding: 0.5em 0;
+  }
+  #theme > button {
+    border-radius: 0;
+    border: 1px solid black;
+    padding: 0.25em 0.5em;
+  }
+  #theme > button:not(:first-of-type) {
+    border-left: none;
+  }
+
+  #theme > button:first-of-type {
+    border-top-left-radius: 3px;
+    border-bottom-left-radius: 3px;
+  }
+
+  #theme > button:last-of-type {
+    border-top-right-radius: 3px;
+    border-bottom-right-radius: 3px;
+  }
+
+  #theme > button {
+    background: #ddd;
+  }
+
+  #theme > button:where(:hover, :focus-within) {
+    background: #ccc;
+  }
+
+  #theme > button.pressed {
+    background: #bbb;
+  }
+
+  #theme > button.pressed:where(:hover, :focus-within) {
+    background: #aaa;
+  }
+
+  #theme > button:active {
+    background: #888;
+  }
+</style>
+<div id="theme">
+  <button id="no-theme">None</button>
+  <button id="classic-theme">Classic</button>
+  <button id="modern-theme">Modern</button>
+</div>
+<script type="module">
+  const diagram = document.getElementById("opentitan-block-diagram");
+  const url = new URL(location);
+  const themes = {
+    "": document.getElementById("no-theme"),
+    "classic": document.getElementById("classic-theme"),
+    "modern": document.getElementById("modern-theme"),
+  };
+
+  for (const [name, button] of Object.entries(themes)) {
+    button.addEventListener("click", () => {
+      for (const [nm, btn] of Object.entries(themes)) {
+        btn.classList.remove("pressed");
+        if (nm) diagram.classList.remove(nm);
+      }
+      button.classList.add("pressed");
+      url.hash = "";
+      if (name) {
+        diagram.classList.add(name);
+        url.hash = `#${name}`;
+      }
+      history.replaceState({}, "", url.href);
+    });
+    if (location.hash.slice(1) == name) {
+      button.click();
+    }
+  }
+</script>
+-->
```

Comment:
Is this commented code deprecated? If so, suggest to remove

Reply:
Removed.